app/src/main/java/app/gamenative/service/battle_net/BattleNetAuthManager.kt

```
@@ -0,0 +1,94 @@
+        }
+    }
+
+    fun loadCookies(context: Context): Map<String, String> {
+        return try {
+            val file = File(getCookiesFilePath(context))
+            if (!file.exists()) return emptyMap()
+            val json = JSONObject(file.readText())
+            val map = mutableMapOf<String, String>()
+            json.keys().forEach { map[it] = json.getString(it) }
+            map
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)